index.js

```diff
@@ -385,7 +396,6 @@ function Argv (processArgs, cwd) {
     return options
   }
 
-  var groups = {}
```

Comment:
FWIW `reset()` seems to be called before this is reached. `groups` should be declared higher up regardless of this PR.